### DISCUSSION TITLE
onion update fail

### DISCUSSION THREAD

**Discussion by Jdura20** [community/community#81949] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i was gifted a miniyoo mini plus yesterday, everything was going great until i tried to install a new onion update, unfortunately for me, the update went through but i lost all my games, and i cant add more games because it says that my sd card is full

**Comment by Akash1134** [comment]
Hi @Jdura20 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by Akash1134** [comment]
#
